libethereum/EthereumHost.cpp

```diff
@@ -383,6 +383,7 @@ EthereumHost::EthereumHost(Host const& _host, BlockChain const& _ch, OverlayDB c
     m_peerObserver = make_shared<EthereumPeerObserver>(m_sync, m_tq);
     m_latestBlockSent = _ch.currentHash();
     m_tq.onImport([this](ImportResult _ir, h256 const& _h, h512 const& _nodeId) { onTransactionImported(_ir, _h, _nodeId); });
+    m_urng = std::mt19937(m_randomSeed());
```

Comment:
Don't keep the random device around. Just init mt19937 here and forget about it.

Reply:
```suggestion
    m_urng = std::mt19937(std::random_device());
```